ledger/committee/src/lib.rs

```diff
@@ -169,14 +169,14 @@ impl<N: Network> Committee<N> {
         self.total_stake().saturating_add(2).saturating_div(3)
     }
 
-    /// Returns the amount of stake required to reach a quorum threshold `(2f + 1)`.
+    /// Returns the amount of stake required to reach a quorum threshold `(N - f)`.
     pub fn quorum_threshold(&self) -> u64 {
         // Assuming `N = 3f + 1 + k`, where `0 <= k < 3`,
-        // then `(2N + 3) / 3 = 2f + 1 + (2k + 2)/3 = 2f + 1 + k = N - f`.
+        // then `2N/3 + 1 = 2f + 1 + (2k + 2)/3 = 2f + 1 + k = N - f`.
```

Comment:
I know this was already there, but can you explain the jump `2f + 1 + (2k + 2)/3 = 2f + 1 + k`? I guess it should be `<=`

Reply:
The `/` is integer division, corresponding to `saturating_div` in the code. Maybe we should add a note clarifying that. (Since k may be 0 or 1 or 2, then (2k+2) div 3 is respectively 0 or 1 or 2, i.e. the same as k.)